pom.xml

```diff
@@ -3720,6 +3720,7 @@
     <plugins>
       <plugin>
         <artifactId>maven-project-info-reports-plugin</artifactId>
+        <version>2.9</version>
```

Comment:
I suppose we make defines for these usually but if one-time use, I think this fine in the pom.

Reply:
Good point! Let me change this to a property.